src/main/java/org/openrewrite/java/testing/junit5/ExpectedExceptionToAssertThrows.java

```diff
@@ -41,16 +54,17 @@
                             "package org.junit.jupiter.api;" +
                             "import java.util.function.Supplier;" +
                             "import org.junit.jupiter.api.function.Executable;" +
-                            "class AssertThrows {\n" +
-                            "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable,Supplier<String> messageSupplier){}" +
-                            "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable,String message){}" +
-                            "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable){}" +
-                            "}"),
+                            "class AssertThrows {" +
+                                "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable,Supplier<String> messageSupplier){}" +
+                                "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable,String message){}" +
+                                "static <T extends Throwable> T assertThrows(Class<T> expectedType, Executable executable){}" +
```

Comment:
I know the previous example didn't have this either, but add a return null; to each method body will prevent compiler errors when parsing. These are silent, but we have seen some performance impact when we get a large number of these.

Reply:
Added the returns